docs/content/posts/2025-03-01-release-2.0.0.md

```diff
@@ -0,0 +1,1639 @@
+
+Furthermore, Flink SQL has introduced specialized syntax for AI models, allowing users to define AI models as easily as defining a Catalog and invoke them like Functions or TableFunctions in SQL statements. Compared to Flink CDC, Flink SQL supports more complex relational data processing logic, seamlessly integrating intricate data processing workflows with AI model invocation. This initiative is currently under active development and refinement.
+
+## Misc
+
+### DataStream V2 API
+
+The DataStream API is one of the two main APIs that Flink provides for writing data processing programs. As an API that was introduced practically since day-1 of the project and has been evolved for nearly a decade, we are observing more and more problems of it. Improvements on these problems require significant breaking changes, which makes in-place refactor impractical. Therefore, we propose to introduce a new set of APIs, the DataStream API V2, to gradually replace the original DataStream API.
+
+In Flink 2.0, we provide the MVP version of the new DataStream V2 API. It contains the low-level building blocks (DataStream, ProcessFunction, Partitioning), context and primitives like state, time service, watermark processing. At the same time, we also provide some high-level extensions, such as window and join. They are more like short-cuts / sugars, without which users can probably still achieve the same behavior by working with the fundamental APIs, but would be a lot easier with the builtin supports.
+
+See [documentations](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream-v2/overview/) for more details
+
+**NOTICE:** The new DataStream API is currently in the experimental stage and is not yet stable, thus not recommended for production usage at the moment. 
+
+### SQL gateway supports application mode
+
+SQL gateway now supports executing SQL jobs in application mode, serving as a replacement of the removed per-job deployment mode.
+
+### SQL Syntax Enhancements
+
+Flink SQL now supports C-style escape strings. See the [documentation](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/queries/overview/#syntax) for more details.
+
+A new `QUALIFY` clause has been added as a more concise syntax for filtering outputs of window functions. Demonstrations on this can be found in the [Top-N](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/queries/topn/) and [Deduplication](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/queries/deduplication/) examples.
+
```

Comment:
One more thing probably is worth mentioning since there is a dedicated `SQL Syntax enhancement` section.

For table function calls it is now possible to use them without `TABLE()` wrapper in `FROM`. Queries with such wrapper will continue working as before.